# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/syntax-highlighting/hotfix.patch.cross
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- syntax-highlighting-6.21.0/src/indexer/CMakeLists.txt.vanilla	2025-12-17 13:51:17.774457500 +0100
+++ syntax-highlighting-6.21.0/src/indexer/CMakeLists.txt	2025-12-17 13:51:33.878804286 +0100
@@ -14,7 +14,7 @@
                         CMAKE_ARGS -DKSYNTAXHIGHLIGHTING_USE_GUI=OFF
                             -DECM_DIR=${ECM_DIR} -DCMAKE_PREFIX_PATH=${NATIVE_PREFIX}
                             -DCMAKE_INSTALL_PREFIX=${CMAKE_CURRENT_BINARY_DIR}
-                            -DQT_MAJOR_VERSION=6
+                            -DQT_MAJOR_VERSION=6 -DCMAKE_C_COMPILER=cc -DCMAKE_CXX_COMPILER=c++
                         INSTALL_COMMAND ""
                         BUILD_BYPRODUCTS ${CMAKE_CURRENT_BINARY_DIR}/native_katehighlightingindexer-prefix/src/native_katehighlightingindexer-build/bin/katehighlightingindexer
                     )
